packages/millicast-sdk/src/types/index.d.ts

```diff
@@ -1463,6 +1511,28 @@ declare module '@millicast/sdk' {
      * @param {Array<String>} mediaIds - mid value of the receivers that are going to be detached.
      */
     unproject(mediaIds: Array<string>): Promise<void>;
+
+    /**
+     * Configure DRM protected stream.
+     * When there are {@link EncryptionParameters} in the payload of 'active' broadcast event, this method should be called
+     */
+    configureDRM(options: DRMOptions);
+
+    /**
+     * Remove DRM configuration for a mediaId
+     */
+    removeDRMConfiguration (mediaId: string);
+
+    /**
+     * Check if there are any DRM protected Track
+     */
+    get isDRMOn(): boolean;
+
+    /** Exchange the DRM configuration between two transceivers
+     *  Make sure both of the transceivers have been used for DRM protected streams
+     */
+    exchangeDRMConfiguration (targetMediaId: string, sourceMediaId: string);
```

Comment:
Question : is this function used with a multi-source DRM stream?

Reply:
yes

Reply:
@dubeyShivank do you have any further questions? Or else please resolve this, thx.